### PR TITLE
Soarian FHIR R4 Practitioner Documentation

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
Review request for Soarian FHIR R4 Practitioner Documentation

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
Practitioner_1
Practitioner_2
Practitioner_3
Practitioner_4
Practitioner_5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/base/individuals/practitioner.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## Search
+
+Search for Practitioner resources that meet specified query parameters:
+
+	GET /Practitioner?:parameters
+
+_Implementation Notes_
+
+*	A `422 Unprocessable Entity` response status code and OperationOutcome are returned when > 499 Practitioners qualify for the search criteria. Fast Healthcare Interoperability Resources (FHIR) applications may prompt the user for more specific search criteria to avoid this possibility.
```
Comment (HRNWriter) [inline content/soarian/r4/base/individuals/practitioner.md:31]: Please add a registered trademark symbol after "Fast Healthcare Interoperability Resources" and "(FHIR)".

**Comment by HRNWriter** [comment]
I added a comment in GitHub. Two registered trademark symbols are needed in the Search Implementation Notes - one after "Fast Healthcare Interoperability Resources" and one after "(FHIR)".

Otherwise all looks good!

Heather Nelson
She/her
Documentation Architect, Product Documentation MLV
***@***.******@***.***> | 610-219-2296
Cerner Corporation | www.cerner.com<http://www.cerner.com>

From: Deepak Gupta ***@***.***>
Sent: Thursday, September 30, 2021 7:33 AM
To: cerner/fhir.cerner.com ***@***.***>
Cc: Nelson, Heather ***@***.***>; Review requested ***@***.***>
Subject: [cerner/fhir.cerner.com] Soarian FHIR R4 Practitioner Documentation (#739)


Description
Review request for Soarian FHIR R4 Practitioner Documentation

PR Checklist

  *   [ ] Screenshot(s) of changes attached before changes merged.
[Practitioner_1]<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F37074481%2F135447334-383902e9-e7e7-40c7-8fce-dc62e2ce4447.JPG&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654453107%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5CTr%2Bf6pKpJfjCsn4PNGAhb2KUjoazdbtZOQvv3xQbM%3D&reserved=0>
[Practitioner_2]<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F37074481%2F135447555-457f3c7a-2127-4c85-9285-c2166ce67fd4.JPG&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654463101%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0wckTUzN3lLb8pnUb%2BzJYOAEeEc3m8Q1uYIxl3S8xEQ%3D&reserved=0>
[Practitioner_3]<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F37074481%2F135447374-74ce8620-6a5b-4d6b-a737-d67689bc6edc.JPG&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654473096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=19ST8vp3ejVPW9PzFKVRxyrhmSHOv0GKNvw4eV1YxT4%3D&reserved=0>
[Practitioner_4]<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F37074481%2F135447381-619f8709-5167-4212-a531-6b7994032f92.JPG&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654473096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=OFlAcLIJtTvT6Hwjt2%2Bk8YCIjcqeoOfvUjdV%2BpD8zg8%3D&reserved=0>
[Practitioner_5]<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F37074481%2F135447390-8d795bec-1011-4489-89b1-cfc76bd8c21d.JPG&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654483089%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=4x2m2opCBj1OqcmwokfdyLnrId0hQUDaEKjYyOP0l30%3D&reserved=0>
  *   [ ] Screenshot(s) of changes attached after changes merged and published.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/cerner/fhir.cerner.com/pull/739<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654483089%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=3eR%2BGwymOlYGokiMlSxX9MSaDcddsUzhwVkOr%2FcQW8s%3D&reserved=0>

Commit Summary

  *   FHIR R4 Soarian Clinicals Documentation: Practitioner Resource<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Fcommits%2F8b9c1d8a53107efcab29e625638337ef635c3e79&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654493087%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=hUlvu5rUv6HgLkq4UgUSo7ME2om0qezbkAGI1PoJCoM%3D&reserved=0>
  *   Merge branch 'soarian_r4_practitioner_write' of https://github.com/cerner/fhir.cerner.com into soarian_r4_practitioner_write<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Fcommits%2F6025d0a23937e87e3fcdfb074f9b63b98edcd5bc&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654493087%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Dj7uwyoHMbLc1qxUgdQA309maHERNKytAtRb34V84%2FQ%3D&reserved=0>
  *   FHIR R4 Practitioner Resource: fixed lint errors<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Fcommits%2F7905ea4c15ad9cecd979856f31adb2d53d03d5a4&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654503077%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=tRdDNyqdL7qAFBCCtv066UtI4GWgf6t63x704os1qBU%3D&reserved=0>
  *   Soarian R4 Practitioner Resource, example crrection<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Fcommits%2F8b8e41ecf4972f1eca25c2cf55d7a43eb6eaddc7&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654503077%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2Fff23sWyw1lsGyU64QE%2BSSA8rqz4W4r57a%2F30UIBgkk%3D&reserved=0>
  *   FHIR R4 Practitioner Resource, .md file correction<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Fcommits%2Fc7cf9200ee32bfaf17af8ac833b6a88fb4748f7d&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654513073%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=W9X4Lvm53duNmdavQ2alR9tNfFKca1cc1UWUcccjF6U%3D&reserved=0>

File Changes

  *   M content/soarian/r4/base.md<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Ffiles%23diff-51f2f236ba486e3d5c03d778bf8ac864ca0017f664f09f84e082225bc7ed13b7&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654523068%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=5yh3bu9nsyawfwtShiVrgbOT9gVCc3SQ8OKVmM2zS%2FM%3D&reserved=0> (3)
  *   M content/soarian/r4/base/individuals.md<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Ffiles%23diff-b6ab17072da130eb3201ca36a99f88e5774b8c4a017fdda0721539feca7de02c&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654523068%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=1hy0fYsWA2XKitfhzxQ5xR9Y5MFEJQ3FgrdPCPskxZw%3D&reserved=0> (3)
  *   A content/soarian/r4/base/individuals/practitioner.md<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Ffiles%23diff-b8da112cc43db39d41b929b09798186916abebe25faa73da4b918b1f6e133d86&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654533065%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=C2AzXa4WY8XHuDBDXcq7IlJVrLoKmvuOWW4z7oKQ6Vo%3D&reserved=0> (104)
  *   M layouts/soarian/r4/sidebar.html<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Ffiles%23diff-c69fcd625bbfe954fb23ad4190cb6e8999a4d54eea65ae5cc181c85aa72f9d83&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654533065%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=sTgqKLrUvvYG6%2FxpW%2B9dx5WMPMQmCgR4J%2F%2FG1bSWsvE%3D&reserved=0> (1)
  *   A lib/resources/example_json/soarian_r4_examples_practitioner.rb<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739%2Ffiles%23diff-54ac34a168743380e97a4aa508546ae4454cb5092c305caeabeae4e75a7106cd&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654543056%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=KCRYQnuP7it%2FcnaTbep6jV2LvK1GyDeUdBHCiBAX1OU%3D&reserved=0> (93)

Patch Links:

  *   https://github.com/cerner/fhir.cerner.com/pull/739.patch<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739.patch&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654543056%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=siJZbCTfq1eGF9l8fEdxxWd3fDVltWKtf7K0EXJPvOI%3D&reserved=0>
  *   https://github.com/cerner/fhir.cerner.com/pull/739.diff<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739.diff&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654553050%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=2lbBb%2FhNRYlyiawCHf1DTpl9cDoURSEdtnFz46y3Bbc%3D&reserved=0>

-
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcerner%2Ffhir.cerner.com%2Fpull%2F739&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654553050%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=v41IbzttlTnwx1ui3Z90SJz9wupUYhBynCcuEpvpFX8%3D&reserved=0>, or unsubscribe<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUTKUSGHLXANUQ4CFRX6MQTUERDFVANCNFSM5FCDMP7A&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654563044%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=m8IP%2FsYPG1Ms6haayeMhs171qMNuJq%2FGYR8vjCMBu68%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654563044%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=eWSHTSxuMC1BcoqE4XobTRNN3HHhkKZCj35dmRJQJ1k%3D&reserved=0> or Android<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cheather.nelson%40cerner.com%7C3c8f896d5af64711c94408d98406047f%7Cfbc493a80d244454a815f4ca58e8c09d%7C0%7C0%7C637685983654573042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Qywy4yQYFf5c3jzQppnJHTxsZaSJ8PiWXthH4wfRS1E%3D&reserved=0>.



CONFIDENTIALITY NOTICE This message and any included attachments are from Cerner Corporation and are intended only for the addressee. The information contained in this message is confidential and may constitute inside or non-public information under international, federal, or state securities laws. Unauthorized forwarding, printing, copying, distribution, or use of such information is strictly prohibited and may be unlawful. If you are not the addressee, please promptly delete this message and notify the sender of the delivery error by e-mail or you may call Cerner's corporate offices in Kansas City, Missouri, U.S.A at (+1) (816)221-1024.

File: content/soarian/r4/base/individuals/practitioner.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+*	The common [errors](#errors) and [OperationOutcomes] may be returned. 
+*	In addition, a `422 Unprocessable Entity`  HTTP response status code and OperationOutcome is returned when > 499 Practitioners qualify for the search criteria.
+
+## Retrieve by ID
+
+List an individual Practitioner resource by its resource ID:
+
+	GET /Practitioner/:id
+
+## Authorization Types
```
Comment (ShrutiNHegde) [inline content/soarian/r4/base/individuals/practitioner.md:73]: Authorization Types needs to be added under Retrieve By ID( Same as Search) not as Header.

File: content/soarian/r4/base/individuals/practitioner.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Request
+
+	GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Practitioner?_id=DF2821C049CC47E6A37C6B796DF4CF1D
+	
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_PRACTITIONER_SEARCH_BY_ID) %>
+
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply content/soarian/r4/base/individuals/practitioner.md:61]: Made changes as suggested

File: content/soarian/r4/base/individuals/practitioner.md (hunk 2 above)
Comment (DeepakGupta93) [reply content/soarian/r4/base/individuals/practitioner.md]: Added Authorization Types under Retrieve by ID

**Review by slvsridhar — APPROVED** [review]

**Review by vf056083 — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]

**Comment by DeepakGupta93** [comment]
After Publish:
image
image
image

**Pull request merged**